alertmanager/config.go

```diff
@@ -16,8 +16,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package alertmanager provides utilities to work with Alertmanager's configuration
 package alertmanager
 
+import (
+	"errors"
+
+	"github.com/mohae/deepcopy"
```

Comment:
`promconfig` is zero-dependency lib, we highlighting this in readme. Existing deps used only in tests, all regular lib files has no external dependencies. Also it seems that `deepcopy` a little bit abandoned.